OrbitCore/FramePointerValidator.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "FramePointerValidator.h"
+
+#include <capstone/capstone.h>
+
+#include "FunctionFramePointerValidator.h"
+#include "OrbitBase/Logging.h"
+
+std::vector<std::shared_ptr<Function>> FramePointerValidator::GetFpoFunctions(
+    std::vector<std::shared_ptr<Function>> functions, std::string file_name,
```

Comment:
Can any of these be passed by const ref?

Reply:
Ah, yes both. Thanks for spotting this.